src/cache.rs

```diff
@@ -59,7 +59,7 @@ struct Transaction2 {
     path_tree: sled::Tree,
     invert_tree: sled::Tree,
     trigram_index_tree: sled::Tree,
-    missing: Vec<(filter::Filter, git2::Oid)>,
+    missing: Vec<(usize, filter::Filter, git2::Oid)>,
```

Comment:
could we give this usize a name? (nesting I suppose)

Reply:
Removed it ;)